spinalcordtoolbox/deepseg/inference.py

```diff
@@ -238,6 +238,7 @@ def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
     # TODO: document this on our intranet (https://intranet.neuro.polymtl.ca/data/deeplearning-models.html#packaging-models)
     if 'orientation' in predictor.dataset_json:
         model_orientation = predictor.dataset_json['orientation']
+        print(f"Model orientation (based on dataset.json): {model_orientation}")
```

Comment:
I added this `print` statement to inform the user that the orientation was successfully read from `dataset.json`. What do think?

Reply:
Yes I think it's a good thing !
Just i suppose we switch the key to "image_orientation" now ?

Reply:
Yes, updated in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4687/commits/e39df18f46fa8c6bb5f065f1e83c622efab10387. The comment is now outdated (thus I resolved it).